### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by axilleas**
Update the readme file to be more clear.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+password
```
Comment (axil42) [inline .gitignore:1]: Cool!

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
 # GitHub Essentials v2
 
 Second revision of the [GitHub Essentials](https://githubessentials) book.
+
+Test change.
```
Comment (axil42) [inline README.md]: Is this needed?

**Review by axil42 — DISMISSED** [review]
Thanks! Just a few remarks :+1:

File: .gitignore (hunk 1 above)
Comment (axilleas) [reply .gitignore]: Thanks

Comment (axilleas) [reply .gitignore:1]: Thanks

File: README.md (hunk 2 above)
Comment (axilleas) [reply README.md]: Good catch, I'll change it!

**Pull request merged**